### ISSUE TITLE
The Mark of Vashj

### ISSUE THREAD

**Issue by jerbookins** [TrinityCore/TrinityCore#21069]
**Description:**
http://www.wowhead.com/quest=10900/the-mark-of-vashj

This quest is provided by Skar'this the Heretic in Heroic Slave Pens.

**Current behaviour:**
Skar'this provides this quest regardless of your level.

**Expected behaviour:**
This quest is ONLY available to level 70 players.

Completing this quest prior to patch 3.0.2 would provide you with the follow up [The Cudgel of Kardesh](http://www.wowhead.com/quest=10901/the-cudgel-of-kardesh) (10901). *Obviously this is impossible* so instead it should provide you with the new version of the quest [The Cudgel of Kardesh](http://www.wowhead.com/quest=13431/the-cudgel-of-kardesh) (13431).

Both versions of these quests are also ONLY available to level 70 players.

**Branch(es):** 3.3.5

**TC rev. hash/commit:** 906a1331b008

**TDB version:** 335.63

**Operating system:** Windows 10

**Comment by offl** [comment]
Try with this:
```sql
--
DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId`=19 AND `SourceGroup` IN (10900);
INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
(19, 0, 10900, 0, 0, 27, 0, 70, 0, 0, 0, 0, 0, "", "Quest 'The Mark of Vashj' requires level 70");
```

**Comment by Killyana** [comment]
This issue is not valid:
SELECT * FROM quest_template WHERE id=10900; -- minlevel=70 (tested and works correctly)
Also the quest giver is spawned on the heroic difficulty only, and to enter this donjon on heroic you must at least be level 70.

**Comment by offl** [comment]
http://www.wowhead.com/quest=10900/the-mark-of-vashj#comments:id=774064

**Comment by offl** [comment]
@Killyana the quest can be obtained when level above 70(this shouldn't happen), not when level below 70. Read the comment above.

**Comment by Killyana** [comment]
It's just a single comment, there's no proof that this quest is only for chars level 70, and it doesn't make sense.

**Comment by offl** [comment]
http://www.wowhead.com/quest=10900/the-mark-of-vashj#comments:id=529420:reply=128387
http://www.wowhead.com/quest=10900/the-mark-of-vashj#comments:id=1505652

> and it doesn't make sense

Well, ok

**Comment by Killyana** [comment]
For this reason we ask for proof in this kind od issues.

**Comment by jerbookins** [comment]
https://us.battle.net/forums/en/wow/topic/7383798668

**Comment by Killyana** [comment]
It's fixed.

**Comment by tkrokli** [comment]
Both MinLevel and MaxLevel requirements?

**Comment by Killyana** [comment]
Killyana added a commit that referenced this issue 5 hours ago
DB/Quest: The Mark of Vashj/The Cudgel of Kar'desh

**Comment by tkrokli** [comment]
Sure, I saw the commit. I presumed it is possible to take the quest before reaching level 70, but I guess that is covered elsewhere.

I also presumed that @jerbookins' comment https://github.com/TrinityCore/TrinityCore/issues/21069#issuecomment-352293247 was based on still missing criteria to make the quest only available at level 70, not 69 or lower (as well as not higher, which I took for granted was already fixed).
